pointgrey_camera_driver/CMakeLists.txt

```diff
@@ -19,14 +19,21 @@ catkin_package(
 # the Point Grey EULA prohibits redistributing the headers or the packages which
 # contains them. Fortunately, we can download the archive directly from their 
 # website during this build process.
-find_library(POINTGREY_LIB flycapture)
+find_library(POINTGREY_LIB NAMES libflycapture.so.2 flycapture)
 if(NOT POINTGREY_LIB)
   message(STATUS "libflycapture not found in system library path")
   include(cmake/DownloadFlyCap.cmake)
   download_flycap(POINTGREY_LIB POINTGREY_INCLUDE_DIR) 
   message(STATUS "libflycapture library: ${POINTGREY_LIB}")
   message(STATUS "libflycapture include: ${POINTGREY_INCLUDE_DIR}")
   include_directories(${POINTGREY_INCLUDE_DIR})  
```

Comment:
What do you need POINTGREY_INCLUDE_DIR for?
It's only used to set the makefile include path to add the non-standard location when we download it.  It's not needed if the library is found b/c the headers are already in the standard location, so the include path is already included in the compile.
If we were redistributing the header files like we are doing with the library then yes we would need to know their location.